set-vars.md

```diff
@@ -0,0 +1,159 @@
+---
+title: SET (session settings)
+summary: The SET statement modifies the current settings for the client session.
+toc: false
+---
+
+The `SET` [statement](sql-statements.html) can modify one of the
+session setting variables.
+
+{{site.data.alerts.callout_danger}}In some cases, client drivers can drop and restart the connection to the server. When this happens, any session configurations made with <code>SET</code> statements are lost. It is therefore more reliable to configure the session in the client's connection string. For examples in different languages, see the <a href="build-an-app-with-cockroachdb.html">Build an App with CockroachDB</a> tutorials.{{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Required Privileges
+
+No [privileges](privileges.html) are required to modify the session settings.
+
+## Synopsis
+
+General syntax:
+
+~~~
+   SET [SESSION] <variable name> = <values...>
+   SET [SESSION] <variable name> TO <values...>
+   SET [SESSION] TIME ZONE <value>
+~~~
+
+The variable name is case insensitive.
+
+Special syntax cases:
+
+| Syntax | Equivalent to |
+|--------|---------------|
+| `SET SESSION CHARACTERISTICS AS TRANSACTION ISOLATION LEVEL ...` | `SET default_transaction_isolation = ...`
+| `SET TIME ZONE ...` | Special syntax because the variable name contains a space. See [`SET TIME ZONE`](#set-time-zone) below.
+
+Note: the `SET` statement for session settings in unrelated to the
+other [`SET TRANSACTION`](set-transaction.html) statement.
+
+## Supported variables
+
+|Variable | Can be viewed with [`SHOW`](show-vars.html)? | Description | Initial value |
```

Comment:
If this is the first mention of `SHOW` in this doc, it needs some explanation as to why you would use it.

Also, I think it would be a little more clear to move `| Can be viewed with [`SHOW`](show-vars.html)? |` to the last column.